runelite-client/src/main/java/net/runelite/client/plugins/herbsack/HerbSack.java

```diff
@@ -0,0 +1,142 @@
+package net.runelite.client.plugins.herbsack;
+
+import com.google.common.collect.ImmutableList;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+import net.runelite.api.ItemID;
+import static net.runelite.client.RuneLite.RUNELITE_DIR;
+
+public class HerbSack
+{
+	private static final File HERBSACK_RECORD_DIR = new File(RUNELITE_DIR, "herbsack");
+	private static final File HERBSACK_PERSISTENCY_FILE = new File(HERBSACK_RECORD_DIR, "herbsack.dat");
```

Comment:
Why are you using external files? Just use config for this, or the db if you really want to, but that's overkill